### DISCUSSION TITLE
Current documentation (manual) is for v0.3.6. Will there be an update?

### DISCUSSION THREAD

**Discussion by almeidainf** [raspberrypi/picamera2#861] [Q&A]
Hi everyone
The Picamera2 Library documentation ([PDF Manual](https://datasheets.raspberrypi.com/camera/picamera2-manual.pdf)) states that it "describes Picamera2 version 0.3.6", while the latest release at the time of this post is 0.3.14. Is there an update planned for the documentation? Alternatively, can I find updated documentation somewhere else?

Thank you!

**Comment by KLRico** [comment]
I'm just a casual user of this project, but it's been my experience that the manual is still accurate and captures the core mechanics. There have been feature updates and additions, like dual encoding, but these additional features are covered pretty well in the examples.
